LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
Strictly speaking for US Law - it's first year of publication https://www.copyright.gov/title17/92chap4.html#401 of the original work and last year of modification. So 2010-2015 would mean we claim first copyright from 2010 but the protection is from 2015 onwards (125 year for the Crown in the UK) as it has been modified.